### ISSUE TITLE
Insufficient tokens for quota 'logging.googleapis.com/read_requests'

### ISSUE THREAD

**Issue by ayj** [istio/test-infra#443]
The following message appeared multiple times in test logs on a test failure.

> W0831 18:03:10.969] * rpc error: code = ResourceExhausted desc = Insufficient tokens for quota 'logging.googleapis.com/read_requests' and limit 'ReadRequestsPerMinutePerProject' of service 'logging.googleapis.com' for consumer 'project_number:450874614208'.

See https://k8s-gubernator.appspot.com/build/istio-prow/pull/istio_pilot/1150/pilot-e2e-smoketest/238/

**Comment by chxchx** [comment]
This is a known issue from StackDriver logging. We tried increasing our read quota through pantheon and turns out we have set to the highest permitted quota. The problem is worse when more than one prow job are hitting the stackdriver API. I will be working on exponential back-off scheme but in the meantime, we are going to fetch the cluster logs using a single thread to mitigate the quota issue, which means a *failed* presubmit might take 2-3 extra minutes to finish.
